doc/themes/the-things-stack/assets/css/theme.scss

```diff
@@ -180,3 +180,5 @@ html.has-navbar-fixed-top {
     min-height: calc(100vh - #{$navbar-height});
   }
 }
+
+@import "highlight.scss";
```

Comment:
Is this and the corresponding file related to the PR? It's completely fine to keep it. Just checking.

Reply:
Good catch. That commit was supposed to be in a future PR.